package.json

```
@@ -1,6 +1,7 @@
 {
   "scripts": {
     "test:unit": "cd react/ && yarn test:ci",
+    "test:e2e": "npx cypress run",
```

```suggestion
    "test:e2e": "cypress run",
```

Acho que com `npx` ele vai baixar o cypress na hora

---

@kevinch provou o contrario:
> Executes <command> either from a local node_modules/.bin, or from a central cache, installing any packages needed in order for <command> to run.